ibm/resource_ibm_iam_account_settings.go

```diff
@@ -0,0 +1,416 @@
+						},
+						"thread_id": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The thread ID of the server instance processing the request.",
+						},
+						"host": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The host of the server instance processing the request.",
+						},
+						"start_time": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The start time of the request.",
+						},
+						"end_time": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The finish time of the request.",
+						},
+						"elapsed_time": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The elapsed time in msec.",
+						},
+						"cluster_name": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The cluster name.",
+						},
+					},
+				},
+			},
+			"restrict_create_service_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: InvokeValidator(accountSettings, restrictCreateServiceId),
+				Description:  "Defines whether or not creating a Service Id is access controlled. Valid values:  * RESTRICTED - to apply access control  * NOT_RESTRICTED - to remove access control  * NOT_SET - to 'unset' a previous set value.",
+			},
+			"restrict_create_platform_apikey": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: InvokeValidator(accountSettings, restrictCreateApiKey),
+				Description:  "Defines whether or not creating platform API keys is access controlled. Valid values:  * RESTRICTED - to apply access control  * NOT_RESTRICTED - to remove access control  * NOT_SET - to 'unset' a previous set value.",
+			},
+			"allowed_ip_addresses": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Defines the IP addresses and subnets from which IAM tokens can be created for the account.",
+			},
+			"entity_tag": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Version of the account settings.",
+			},
+			"mfa": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: InvokeValidator(accountSettings, mfa),
+				Description:  "Defines the MFA trait for the account. Valid values:  * NONE - No MFA trait set  * TOTP - For all non-federated IBMId users  * TOTP4ALL - For all users  * LEVEL1 - Email-based MFA for all users  * LEVEL2 - TOTP-based MFA for all users  * LEVEL3 - U2F MFA for all users.",
+			},
+			"if_match": &schema.Schema{
```

Comment:
if_match is required but you are not setting in testcase did the testcase executed or worked?

Reply:
Fixed.